gdcdictionary/schemas/molecular_test.yaml

```diff
@@ -980,6 +1028,33 @@ properties:
       - Total Protein
       - Unknown
       - Not Reported
+      - Epstein-Barr Virus
+      - Human Papillomavirus
+      - HPV-E6/E7
+      - Circulating Tumor Cells
+      - B-cell genotyping (BLGSP)
```

Comment:
Is the "BLGSP" just signifying which project asked for the change? @tlicht3

Reply:
yeah - this also needs updated.